packages/eslint-plugin-fbtee/src/index.tsx

```diff
@@ -10,13 +11,15 @@ export const meta = {
 export const rules = {
   'no-empty-strings': noEmptyStringsRule,
   'no-unhelpful-desc': noUnhelpfulDesc,
+  'no-unwrapped-strings': noUnwrappedStrings,
 };
 
 export const configs = {
   strict: {
     rules: {
       '@nkzw/fbtee/no-empty-strings': 2,
       '@nkzw/fbtee/no-unhelpful-desc': 2,
+      '@nkzw/fbtee/no-unwrapped-strings': 2,
```

Comment:
I'm not sure about this rule being included as an error by default. Since it doesn't have an autofix, it's going to be annoying for incremental adoption and most people would have to disable it until they complete the migration.

How about we export two configs:
* Enabled by default in strict.
* Disabled by default in a new "recommended" config.

+ update the README to explain the differences.

What do you think?

Reply:
Makes sense, do you mean in the main README or a specific one just for the eslint plugin?

Reply:
Main README in the ESLint section for now. The main README gets copied into each package's root directory when publishing.

Reply:
Missed that section somehow! It's updated now